src/app/cli/src/error.rs

```diff
@@ -213,6 +214,12 @@ impl From<InternalError> for CLIError {
     }
 }
 
+impl From<ReadlineError> for CLIError {
```

Comment:
Where do we use this conversion?

I see that the error from `exec_from_repl` is mapped explicitly.

If this is not necessary any more - we could also remove the explicit dependency on `rustyline` from `Cargo.toml`.